src/utils/verifySignature.ts

```diff
@@ -6,9 +6,10 @@ export function verifySignature(
   payload: string,
   signature: string,
   secret: string,
-  createDigest = (i) => i
+  createDigest = (i) => i,
+  method: 'sha1' | 'sha256' = 'sha1'
```

Comment:
Can we default to the stronger algorithm, `sha256`?

Reply:
The function this function calls defaults to sha1, I can change all existing usages that rely on that default to be explicit and change the default to sha256.